core/executor/src/wasm_executor.rs

```diff
@@ -174,6 +174,10 @@ impl_function_executor!(this: FunctionExecutor<'e, E>,
 				HexDisplay::from(&key)
 			);
 		}
+		let storage_key = ChildStorageKey::from_vec(storage_key)
+			.ok_or_else(||
+				UserError("ext_set_child_storage: child storage key is invalid")
```

Comment:
We do this a lot.  Should we make a convienience function?

Reply:
Yeah, but unfortunately I couldn't come up with such a function that it provides enough offset (i.e. convenient to use and also provides function names).

Reply:
Could we use a macro, or pass the string explicitly?